GameBoard/Views/Shared/Components/RemoveFromGameEventForm/RemoveFromGameEventFormViewComponent.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using GameBoard.Models.GameEvent;
+using GameBoard.Models.GameEventInviteForm;
+using Microsoft.AspNetCore.Mvc;
+
+namespace GameBoard.Views.Shared.Components.RemoveFromGameEventForm
+{
+    public class RemoveFromGameEventFormViewComponent : ViewComponent
+    {
+        public IViewComponentResult Invoke(int gameEventId, string gameEventName, string userName) =>
+            View("RemoveFromGameEventForm", new RemoveFromGameEventViewModel
+            {
+                GameEventId = gameEventId,
+                GameEventName = gameEventName,
+                UserName = userName
+            });
```

Comment:
There is no reason for this component to accept three arguments and convert them into an object right away. It could just as well accept a single `RemoveFromGameEventViewModel`.

Reply:
It seems not to work as I change this. I see you did the same thing in the GameEventInviteFromViewComponent.

Reply:
Fixed.